layouts/partials/slides/columns.html

```diff
@@ -28,14 +28,14 @@
 	{{ end }}
 	{{ if not $columnsCount }}
 		<!-- default value doesn't exist in config.yaml -->
-		{{ $columnsCount = 3 }} 
+		{{ $columnsCount = 3 }}
 	{{ end }}
 {{ end }}
 
 {{ $posts := slice }}
 {{ if .ctx.Site.Taxonomies.tags }}
 	{{ $posts = index .ctx.Site.Taxonomies.tags .ctx.Params.Title }}
-	{{ $posts = sort $posts ".Params.weight" "desc" }}
+	{{ $posts = sort $posts ".Params.weight" "asc" }}
```

Comment:
Let's keep this as desc as discussed at #5

Reply:
I revert back to the descending order.